modules/toyvolume/rendering/renderabletoyvolume.cpp

```diff
@@ -196,7 +198,9 @@ void RenderableToyVolume::update(const UpdateData& data) {
                 std::pow(10.f, static_cast<float>(_scalingExponent))
         );
 
-        _raycaster->setColor(_color);
+        glm::vec4 color{ glm::vec3(_color), _opacity };
```

Comment:
Isn't this a repeat of line 144?

Reply:
Yes, but my impression is that is is needed, since the color value is used to set the color for the raycaster at both of these places?